frontend/src/components/TokenSelector.jsx

```diff
@@ -11,25 +11,32 @@ const Tokens = [
   { id: 'daiOption', component: <DAI />, label: 'DAI' },
 ];
 
-const TokenSelector = ({ setSelectedToken }) => (
-  <div className='form-token'>
-    {Tokens.map((token) => (
-      <div className='token-card flex' key={token.id}>
-        <input
-          type='radio'
-          id={token.id}
-          name='token-options'
-          value={token.label}
-          onChange={() => setSelectedToken(token.label)}
-        />
-        <label htmlFor={token.id}>
-          <h5>
-            {token.component} {token.label}
-          </h5>
-        </label>
-      </div>
-    ))}
-  </div>
-);
+const TokenSelector = ({ setSelectedToken }) => {
+  const [tokenSelect, setTokenSelect] = useState('STRK');
```

Comment:
why do we have this state? we already have `selectedToken`

Reply:
but we already have `selectedToken` in `Form.jsx` that can be passed to `TokenSelector` component

Reply:
still unresolved after latest changes